research/object_detection/core/model.py

```diff
@@ -39,6 +39,17 @@
 Output classes are always integers in the range [0, num_classes).  Any mapping
 of these integers to semantic labels is to be handled outside of this class.
 
+Images are resized in the `preprocess` method. All of `preprocess`, `predict`,
+and `postprocess` should be stateless.
```

Comment:
Define "stateless" (this probably means that the model class should not store any data between calls IIUC).

Maybe reentrant is a better term (we can interrupt the preprocess->predict->postprocess flow for a given batch of images with a different batch of images, then resume the process on the first batch.)